docs/Community/Governance.md

```diff
@@ -0,0 +1,161 @@
+
+All SIG and WG leaders are responsible for their groups' creation of minutes from
+any meetings which they may hold, ensuring that the meetings are open and inclusive, and
+that decisions they make are fair and transparent.
+
+If a private discussion pertains to an open issue or PR, and results in a
+decision, that item should be updated and reference given to the conversation.
+
+Project-wide technical discussions are currently handled in
+[SIG-Architecture](sig-architecture/README.md) meetings. This SIG is comprised
+of the Leads of all other SIGs and functions analogously to a "technical
+steering committee" in other projects.
+
+Issue/PR Review Process
+-----------------------
+
+The community strives to triage and respond to all incoming issues within one
+week. If you haven't heard anything by then, or an issue or PR is not getting
+any response within a reasonable time after it is triaged, please feel free to
+remind us with a ping on the thread.
+
+Security issues should be reported through a separate channel, and
+will receive a response within 24 hours. See [Reporting Security
+Issues](Contributing.md#reporting-security-issues).
+
+Committers and Contributors
+---------------------------
+
+At present, we maintain a list of [Project Committers](Committers.md) along with
+descriptions of their knowledge areas.
+
+This will be updated and replaced with a more scalable approach to
+commit-rights management that integrates with SIG/WG governance.
+
+**Proposal** The following roles would be sufficient for SIGs and WGs to self-govern.
+
+- **Chair**: the primary point of contact for a SIG or WG, responsible for the processes within the SIG; a tie-breaker in votes within the SIG, if needed
+- **Approver**: members of the SIG who are empowered to approve code changes
+- **Reviewer**: members of the SIG whose knowledge is respected by Committers, and whom are frequently asked to provide code reviews for patches
+- **Member**: a voting member of a working group (role does not apply to SIGs)
+- **Emeritus**: a previously-active member of a SIG or WG, who has no current rights but whose contributions the community wishes to honor, and who may receive a fast-track back to privileges should they become active in this group again
+
+Accepting Contributions
+-----------------------
+
+SIGs will generally accept changes that improve the project and that align with the
+[Open Enclave roadmap](https://github.com/openenclave/openenclave/projects).
+
+All contributions are expected to be reviewed by at least one *Reviewer* or
+*Approver* other than the contributor, and have no significant objections from
+other *Reviewers* or *Approvers*, within the same SIG. Each SIG may,
+optionally, establish its own review criteria, as outlined in that SIG's
+charter.
+
+If a change requires the approval of multiple SIGs (e.g., because it affects
+code in both SIG domains), then approval from both SIGs is required.
```

Comment:
So changes to the public API surface, if it related to attestation, needs approval from SIG-Architecture?

Reply:
My view: yes, since SIG-Arch is responsible for checking for adherence to the API guidelines and consistency across the full public API surface area